content/src/anytime-action-tasks/carnival-ride-supplemental-modification.md

```diff
@@ -4,10 +4,6 @@ notesMd: "[Muni planning
   Av4xC1g/edit)"
 filename: carnival-ride-supplemental-modification
 name: Modify Your Carnival or Amusement Ride
-industryIds:
```

Comment:
Needed to remove this to only have the anytime action appear with profile question - cleared it with Micah